library/general/test/greasemonkey_test.rb

```diff
@@ -0,0 +1,151 @@
+#! /usr/bin/env rspec
+require_relative "test_helper"
+require "ui/greasemonkey"
+
+describe "UI::Greasemonkey" do
+  RSpec.shared_examples "a Greasemonkey method" do |mname|
+    it "transforms its argument properly" do
```

Comment:
it should better document what happen like `it "transforms GreaseMonkey term to UI term"`